conf/examples/without_presistent_monitoring.yaml

```diff
@@ -0,0 +1,7 @@
+---
+ENV_DATA:
+  # disable persistent monitoring (reconfiguration of OCP Prometheus and alert
+  # manager to use OCS backed PVs) but make sure monitoring is enabled
+  monitoring_enabled: true
+  cluster_namespace: 'openshift-storage'
```

Comment:
This is already the default value so you can remove this line

Reply:
Line dropped.